### DISCUSSION TITLE
Add folders to 'Your stars' page

### DISCUSSION THREAD

**Discussion by jac0-b** [community/community#4470] [Other Feature Feedback, Questions, & Ideas]
The 'Your stars' page is not useful for finding a starred repository in it's current state. It would be much more useful to let us create folders of repos in this page instead of having to click through multiple pages of repos just to find the right one.

If you like this idea consider upvoting it ⬆ instead of thumbs up 👍

**Comment by cromatikap** [comment]
Yes, a way to sort starred repos into personal categories would be nice.
I also found [this issue](https://github.com/isaacs/github/issues/1233) with 12 up-votes asking for the same feature.
